packages/react/src/auto/shadcn/inputs/relationships/SelectedRelatedRecordTags.tsx

```diff
@@ -0,0 +1,58 @@
+import { XIcon } from "lucide-react";
+import React from "react";
+import { getRecordsAsOptions } from "../../../hooks/useRelatedModel.js";
+import type { OptionLabel } from "../../../interfaces/AutoRelationshipInputProps.js";
+import type { ShadcnElements } from "../../elements.js";
+
+export const makeSelectedRecordTags = ({ Badge, Button }: Pick<ShadcnElements, "Badge" | "Button">) => {
+  function SelectedRecordTags(props: {
+    selectedRecords: Record<string, any>[];
+    optionLabel: OptionLabel;
+    onRemoveRecord: (record: Record<string, any>) => void;
+  }) {
+    const { selectedRecords, optionLabel, onRemoveRecord } = props;
+
+    const options = getRecordsAsOptions(selectedRecords, optionLabel);
+
+    if (!options.length) {
+      return null;
+    }
+
+    //TODO: why is the key not the id?
```

Comment:
Can you explain?

Reply:
once again, me asking myself questions to come back to. Please ignore @airhorns